src/main/webapp/WEB-INF/web.xml

```diff
@@ -5,18 +5,37 @@
          xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee
                 http://java.sun.com/xml/ns/j2ee/web-app_2_4.xsd">
 
-    <display-name>UnpaywallMirror</display-name>
+    <display-name>OpenAccessMirror</display-name>
 
     <servlet>
         <servlet-name>UnpaywallApi</servlet-name>
-        <servlet-class>unpaywall.Api</servlet-class>
+        <servlet-class>oamirror.Api</servlet-class>
         <load-on-startup>1</load-on-startup>
+        <init-param>
+            <param-name>datasource</param-name>
+            <param-value>unpaywall</param-value>
+        </init-param>
+    </servlet>
+
+    <servlet>
+        <servlet-name>CrossrefApi</servlet-name>
+        <servlet-class>oamirror.Api</servlet-class>
+        <load-on-startup>1</load-on-startup>
+        <init-param>
+            <param-name>datasource</param-name>
+            <param-value>crossref</param-value>
+        </init-param>
     </servlet>
 
     <servlet-mapping>
         <servlet-name>UnpaywallApi</servlet-name>
         <url-pattern>/v2/*</url-pattern>
     </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>CrossrefApi</servlet-name>
+        <url-pattern>/works/*</url-pattern>
```

Comment:
*Maybe* more clear as e.g. `/unpaywall/v2/*` and `/crossref/works/*`? (Just a hunch, I know to little about the domain differences.)

Reply:
The idea is/was to use exactly the same path that the original services use, so you only have the change the domain part of the url when swithching from real source to mirror or back.
There's also the slight annoyance that if we change the paths, a corresponding fix/release for swepub would also be needed to correct the path there :(

Reply:
Kinda figured that. Just wanted a more cohesive proposal than "perhaps call the new one /crossref/works/*". Of course, an alias for the existing `/v2/*` could work, but for was is basically an internal service, there is no need to clarify/disambiguate here now. If we add 2 or three more, perhaps.

Reply:
I agree!